homeassistant/components/agent_dvr/alarm_control_panel.py

```diff
@@ -41,13 +39,13 @@ async def async_setup_entry(
 class AgentBaseStation(AlarmControlPanelEntity):
     """Representation of an Agent DVR Alarm Control Panel."""
 
-    _attr_icon = ICON
     _attr_supported_features = (
         AlarmControlPanelEntityFeature.ARM_HOME
         | AlarmControlPanelEntityFeature.ARM_AWAY
         | AlarmControlPanelEntityFeature.ARM_NIGHT
     )
     _attr_has_entity_name = True
+    _attr_translation_key = "alarm_control_panel"
```

Comment:
Hmmz shouldn't this be using the default alarm control panel icon instead?

../Frenck

Reply:
I dont have one so i did not know that existed :)